config/authorino/rbac/service_viewer_role.yaml

```
@@ -0,0 +1,20 @@
+# permissions for end users to view services.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: service-viewer-role
```

Same as https://github.com/Kuadrant/authorino-operator/pull/1/files#r740874424

```suggestion
  name: authconfig-viewer-role
```